dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/datasource/SpringConnectionFactory.java

```diff
@@ -58,12 +56,16 @@ public SqlSessionFactory sqlSessionFactory(DataSource dataSource) throws Excepti
         configuration.setCacheEnabled(false);
         configuration.setCallSettersOnNulls(true);
         configuration.setJdbcTypeForNull(JdbcType.NULL);
-        configuration.addInterceptor(paginationInterceptor());
+        configuration.setObjectWrapperFactory(new MybatisMapWrapperFactory());
+
+        MybatisPlusInterceptor mybatisPlusInterceptor = new MybatisPlusInterceptor();
+        mybatisPlusInterceptor.addInnerInterceptor(new PaginationInnerInterceptor());
+        mybatisPlusInterceptor.addInnerInterceptor(new OptimisticLockerInnerInterceptor());
```

Comment:
Does it need to add this optimistic lock plugin? This plugin requires the object to have a @version field to take effect.

Reply:
You are right, i will remove this.